MsApplicationPkg/SecureBootRecovery/SecureBootRecovery.c

```diff
@@ -48,6 +97,210 @@ StatusToHexString (
   return StatusString;
 }
 
+/**
+ * Prints an error message to the console
+ * @param[in] Message     The message to print
+ * @param[in] Status      The EFI_STATUS to print
+*/
+VOID
+PrintError (
+  CHAR16      *Message,
+  EFI_STATUS  Status
+  )
+{

```

Comment:
I suggest at least an assert to simplify debugging in case of a bad value given to the function.

```suggestion
{
  ASSERT (Message != NULL);
```

Reply:
If you're trying to completely avoid using `DebugLib`, then just return on NULL.